### PR TITLE
Fix discount announcement [MAILPOET-1590]

### PR DESCRIPTION
**Pull request by pavel-mailpoet**

### CODE REVIEW

File: lib/Util/Notices/PermanentNotices.php [hunk 1]
```diff
@@ -32,7 +32,11 @@ public function init() {
       Menu::isOnMailPoetAdminPage()
       && $_GET['page'] !== 'mailpoet-welcome-wizard'
     );
-    $this->discounts_announcement->init(empty($_GET['page']) && is_admin());
+    $this->discounts_announcement->init(
+      empty($_GET['page'])
+      && is_admin()
+      && strpos($_SERVER['REQUEST_URI'], 'wp-admin/index.php') !== false
```
Comment (amine-mp) [inline lib/Util/Notices/PermanentNotices.php]: This works when the URL is `/wp-admin/index.php` but does not when it's `/wp-admin/`

File: views/premium.html [hunk 2]
```diff
@@ -5,10 +5,15 @@
...
-    <h1><%= __('Discounts up to 40% for a limited time') %></h1>
+    <h1><%= __('Discounts up to 50% for a limited time') %></h1>
     <p><%= __('Don’t miss out on our only sale of the year which ends on November 30.') %></p>
     <p><%= __('There’s never been a better time to change gears.') %></p>
-    <a href=" https://account.mailpoet.com/?s=<%= subscriber_count %>&utm_source=plugin&utm_medium=premium&utm_campaign=discount">Save up to 40%</a>
+    <a
+      href="https://account.mailpoet.com/?s=<%= subscriber_count %>&utm_source=plugin&utm_medium=premium&utm_campaign=discount"
+      class="button button-primary"
+    >
+      Save up to 40%
```
Comment (amine-mp) [inline views/premium.html]: This also should be "50%"

File: views/premium.html [hunk 3]
```diff
@@ -5,10 +5,15 @@
...
   <% if display_discount %>
   <div class="mailpoet-discount-container">
-    <h1><%= __('Discounts up to 40% for a limited time') %></h1>
+    <h1><%= __('Discounts up to 50% for a limited time') %></h1>
     <p><%= __('Don’t miss out on our only sale of the year which ends on November 30.') %></p>
     <p><%= __('There’s never been a better time to change gears.') %></p>
-    <a href=" https://account.mailpoet.com/?s=<%= subscriber_count %>&utm_source=plugin&utm_medium=premium&utm_campaign=discount">Save up to 40%</a>
+    <a
+      href="https://account.mailpoet.com/?s=<%= subscriber_count %>&utm_source=plugin&utm_medium=premium&utm_campaign=discount"
+      class="button button-primary"
```
Comment (amine-mp) [inline views/premium.html]: I am not sure this is the design we want. What do you think @kgjerstad ?

dd

Comment (pavel-mailpoet) [reply views/premium.html]: Well, this is what Quentin wrote to specification...

File: views/premium.html (hunk 2 above)
Comment (pavel-mailpoet) [reply views/premium.html]: Should this also be translated?

Comment (amine-mp) [reply views/premium.html]: Yes.

File: views/premium.html (hunk 3 above)
Comment (kgjerstad) [reply views/premium.html]: I suggest we copy the style of the "Purchase Now" button at the bottom of the same page: https://d.pr/i/HHlq36

**Pull request merged**